docusaurus/docs/Android/03-ui/04-message-components/02-message-list.mdx

```diff
@@ -461,9 +475,10 @@ The predicate has to return `true` for the items that you _do_ want to display i
 2. Write your own logic for creating ViewHolders. 
 3. Create a new factory instance and set it on `MessageListView`.
 
-Let's consider an example when we want to create custom ViewHolders for messages that came from other users less than 24 hours ago. The result should look like this:
+Let's consider an example in which we want to create a custom ViewHolder for messages sent by other users less than 24 hours ago. The result should look like this:
 
-![](../../assets/message_list_custom_vh_factory.png)
+| ![](../../assets/message_list_custom_vh_factory.png) |
+| --- |
 
 1. Add a new layout called `today_message_list_item.xml`:
```

Comment:
What's slightly confusing me is that this isn't the default formatting.

If you copy paste your example into AS and then auto format it, it will produce the result shown in the docs.